lib/state_of_the_nation.rb

```diff
@@ -30,6 +30,27 @@ def until(finish_key)
           (finish.blank? || round_if_should(finish) > round_if_should(time)) && round_if_should(start) <= round_if_should(time)
         end
 
+        define_method "active_in_interval?" do |interval_start, interval_end|
```

Comment:
`active?` returns `true` when the start time of record is equal to the time. Do we need to the same for the boundaries of the interval here as well?

Reply:
Matches the query string as well.

https://github.com/intercom/state_of_the_nation/blob/cd9edca217f4ac1ed993d47922fb8bc4d59d7604/lib/state_of_the_nation/query_string.rb#L10

Reply:
Whether stuff is active at boundaries is something I thought about when writing the code and writing tests. My assumption was that SOTN doesn't raise validation errors if one record ends at time_x and a subsequent record starts at the same time_x, that they wouldn't be deemed active. 

However, I agree that we should match the behaviour of `active?` here so still update to comply

Reply:
I actually really struggled to find a case where a record was deemed active but we were missing it. The only cases I could find was for the interval start and end being the same as the record start which I added as a case of it's own.